backend/services/auth_service.py

```
@@ -128,6 +133,110 @@ def login_with_password(username: str, password: str):
+        with pool.connection() as conn, conn.cursor(row_factory=dict_row) as cur:
+            cur.execute(
+                """
+                SELECT user_id, expires_at, used
+                FROM public.password_reset_tokens
+                WHERE token = %s
+                LIMIT 1
+                """,
+                (token,),
+            )
+            row = cur.fetchone()
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)